### PR TITLE
Modified pom file to create a tar.gz file for deployment

### PR DESCRIPTION
**Pull request by naderak**

### CODE REVIEW

**Review by danege — COMMENTED** [review]
The assembly-plugin is downgraded from 3.1.1 to 2.6, this is why the automatic build fails.
The compiler-plugin is downgraded from 3.8.1 to 3.8.0 and source and target from v 11 to 1.8

File: pom.xml [hunk 1]
```diff
@@ -188,27 +228,76 @@
...
+                    <version>3.5.1</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-source-plugin</artifactId>
+                    <version>3.0.1</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>2.22.0</version>
+                </plugin>
```
Comment (tokee) [inline pom.xml:297]: These are already defined in the parent POM and should not be defined in the project, unless there is a special need. The parent POM can be inspected at https://sbprojects.statsbiblioteket.dk/nexus/service/local/repositories/releases/content/dk/statsbiblioteket/sbprojects/sbprojects-parent/17/sbprojects-parent-17.pom

File: pom.xml [hunk 2]
```diff
@@ -143,43 +143,83 @@
...
-                    </descriptors>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+
+            <!-- create the deployment layout-->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>appassembler-maven-plugin</artifactId>
```
Comment (tokee) [inline pom.xml:173]: This is an advanced artifact builder. We'll have to discuss whether we should go this way or stick to the simpler artifact building.

**Review by tokee — CHANGES_REQUESTED** [review]

**Comment by tokee** [comment]
This has been superseded by #28 which packaged the project the same way as the template that is to be used at KB.

**Pull request closed without merging**